src/interface/iimpl.rs

```diff
@@ -292,16 +292,33 @@ impl IfacePair {
     pub fn with(iface: Iface, iimpl: IfaceImpl) -> IfacePair { IfacePair { iface, iimpl } }
 
     pub fn iface_id(&self) -> IfaceId { self.iface.iface_id() }
+
     pub fn impl_id(&self) -> ImplId { self.iimpl.impl_id() }
+
     pub fn global_type(&self, name: &FieldName) -> Option<GlobalStateType> {
         self.iimpl.global_type(name)
     }
+
+    pub fn global_name(&self, global_type: &GlobalStateType) -> Option<&FieldName> {
+        self.iimpl.global_name(global_type.to_owned())
```

Comment:
This type is `Copy`, thus there is no need for `to_owned()`. Just change the argument to `global_type: GlobalStateType`

Reply:
done